tests/test_workers/test_pkg_managers/test_general_js.py

```diff
@@ -11,15 +11,23 @@
 from cachito.errors import CachitoError
 from cachito.workers.errors import NexusScriptError
 from cachito.workers.pkg_managers import general_js
+from cachito.workers.paths import RequestBundleDir
 
 
 @mock.patch("tempfile.TemporaryDirectory")
 @mock.patch("cachito.workers.pkg_managers.general_js.generate_and_write_npmrc_file")
-@mock.patch("cachito.workers.pkg_managers.general_js.RequestBundleDir")
```

Comment:
The problem with not mocking this is that certain directories on your system get created and don't get cleaned up due to this:
https://github.com/release-engineering/cachito/blob/4223f878f723504cc7833309b68b37ccfac14f62/cachito/workers/paths.py#L24-L33

My suggestion is either continue mocking it, or use the `tmpdir` pytest fixture and patch the worker config of `cachito_bundles_dir` to use the path assigned by `tmpdir`.